### PR TITLE
Added a stub entry for a FAQ page for later population as required.

### PR DESCRIPTION
**Pull request by Peter-Easton**

### CODE REVIEW

File: static/faq.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+<!--Site footer below. Do not edit below this line.--> 
+        </div>
+        <footer>
+            <a href="/"><img src="https://grapheneos.org/logo.png" width="512" height="512" alt=""/>GrapheneOS</a>
+            <ul id="social">
+                <li><a href="https://twitter.com/GrapheneOS">Twitter</a></li>
+                <li><a href="https://github.com/GrapheneOS">GitHub</a></li>
+            </ul>
+        </footer>
+        <script type="application/ld+json">
```
Comment (thestinger) [inline static/faq.html]: Can you leave out this application/ld+json part? It's only meant for the contact page.

Comment (Peter-Easton) [reply static/faq.html]: Whoops, sure. Pushing it now.

File: static/faq.html [hunk 2]
```diff
@@ -71,6 +71,7 @@ <h2 id="table-of-contents">
 <!--Content below -->
 
 <!--Site footer below. Do not edit below this line.--> 
+        </div>
```
Comment (thestinger) [inline static/faq.html:74]: It seems you accidentally added a div here. Also, can you squash these into a single commit? Use `git rebase -i PARENT_COMMIT` (the one before your first commit) and then switch this 2nd commit from pick to squash. Do not reorder the lines, as that reorders commits.

Comment (Peter-Easton) [reply static/faq.html:74]: Whoops. On it now.

**Pull request merged**